processing/src/main/java/io/druid/segment/QueryableIndexStorageAdapter.java

```diff
@@ -182,8 +185,10 @@ public DateTime getMaxIngestedEventTime()
       actualInterval = actualInterval.withEnd(dataInterval.getEnd());
     }
 
+    Filter[] filters = Filters.partitionWithBitmapSupport(filter);
+
     final Offset offset;
-    if (filter == null) {
+    if (filters[0] == null) {
```

Comment:
I think the previous method can just return null and have a null check here. It'll be easier for ppl to read later on.